README.md

```diff
@@ -17,3 +17,32 @@ looks like this:
+
+```sh
+docker run --name zk-1 -e zk_id=1 -e zk_server.1=172.16.42.101:2888:3888 -e zk_server.2=172.16.42.102:2888:3888 -e zk_server.2=172.16.42.103:2888:3888 -p 2181:2181 -p 2888:2888 -p 3888:3888 confluent/zookeeper
+docker run --name zk-2 -e zk_id=2 -e zk_server.1=172.16.42.101:2888:3888 -e zk_server.2=172.16.42.102:2888:3888 -e zk_server.2=172.16.42.103:2888:3888 -p 2181:2181 -p 2888:2888 -p 3888:3888 confluent/zookeeper
+docker run --name zk-3 -e zk_id=3 -e zk_server.1=172.16.42.101:2888:3888 -e zk_server.2=172.16.42.102:2888:3888 -e zk_server.2=172.16.42.103:2888:3888 -p 2181:2181 -p 2888:2888 -p 3888:3888 confluent/zookeeper
+docker run --name kafka-1 -e KAFKA_BROKER_ID=1 -e KAFKA_ZOOKEEPER_CONNECT=172.16.42.101:2181,172.16.42.102:2181,172.16.42.103:2181 -p 9092:9092 confluent/kafka
+docker run --name kafka-2 -e KAFKA_BROKER_ID=2 -e KAFKA_ZOOKEEPER_CONNECT=172.16.42.101:2181,172.16.42.102:2181,172.16.42.103:2181 -p 9092:9092 confluent/kafka
+```
+
+## Changing settings
+The images support using environment variables via the Docker `-e | --env` flags for setting various settings in the respective images. For example:
+
+  - For the Zookeeper image use variables prefixed with `zk_` with the variables expressed exactly as how they would appear in the `zookeeper.properties` file. As an example, to set `syncLimit` and `server.1` you'd run `docker run --name zk -e zk_syncLimit=2 -e zk_server.1=localhost:2888:3888 confluent/zookeeper`.
+
+  - For the Kafka image use variables prefixed with `KAFKA_` with an underscore (`_`) separating each word instead of periods. As an example, to set `broker.id` and `offsets.storage` you'd run `docker run --name kafka --link zookeeper:zk -e KAFKA_BROKER_ID=2 -e KAFKA_OFFSETS_STORAGE=kafka confluent/kafka`.
+
+  - For the Schema Registry image use variables prefixed with `SR_` with an underscore (`_`) separating each word instead of periods. As an example, to set `kafkastore.topic` and `debug` you'd run `docker run --name schema-registry --link zookeeper:zk --link kafka:kafka -e SR_KAFKASTORE_TOPIC=_schemas -e SR_DEBUG=true confluent/schema-registry`.
+
+  - For the Kafka REST Proxy image use variables prefixed with `RP_` with an underscore (`_`) separating each word instead of periods. As an example, to set `id` and `zookeeper_connect` you'd run `docker run --name rest-proxy --link sr:schema_registry --link zookeeper:zk -e RP_ID=2 -e RP_ZOOKEEPER_CONNECT=192.168.1.101:2182 confluent/rest-proxy`.
```

Comment:
I think here is where the change from one of the other PRs shows up, which results in the `SR_` vs. `SCHEMA_REGISTRY_` naming. However, a) I think this is reversed (given the previous examples such as using --zookeeper name and then --link zookeeper:zk and b) we don't have consistency in the naming/linking with the earlier part of the README.

Is the primary benefit of the renaming just avoiding a bit of verbosity in the env variables? I'm hesitant to lose the clarity of the full names given that most of the time we expect people to use the defaults, or if you're doing a lot of overrides the slight verbosity shouldn't be much of a problem. Then again, I'm not really familiar with the tradeoffs/difficulties with using this type of Docker config in production, so let me know if I'm missing something important.